lib/gui/modules/drive-scanner.js

```diff
@@ -83,6 +108,12 @@ const scan = () => {
   return sdk.scan({
     standard: {
       includeSystemDrives: settings.get('unsafeMode')
+    },
+    usbboot: {
+      files: {
+        'bootcode.bin': USBBOOT_BOOTCODE_BIN_BUFFER,
+        'start.elf': USBBOOT_START_ELF_BUFFER
```

Comment:
I don't know how likely they are to change, but maybe we could use constants here in case these names change?

    files: {
      [BOOTCODE_BIN]: ...,
      [START_ELF]: ...
    }

Comes with the added bonus of making it seem more like there are bins and elves involved.

Reply:
Haha, the thing is that the files you pass here depends on what you want to accomplish. We pass these for our use case, but we could be passing different ones depending on our use case.